File: TransCoder_tokenized_test_set_functions/python/CONVERT_STRING_BINARY_SEQUENCE.py
def strToBinary ( s ) :
    bin_conv = [ ]
    for c in s :
        ascii_val = ord ( c )
        binary_val = bin ( ascii_val )
        bin_conv . append ( binary_val [ 2 : ] )
    return ( ' ' . join ( bin_conv ) )

File: results/java_codex/CONVERT_STRING_BINARY_SEQUENCE.java
void strToBinary ( string s ) {
  int n = s . length ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    int val = s [ i ] - '0';
    string bin = "";
    while ( val > 0 ) {
      if ( val % 2 == 1 ) {
        bin += '1';
      }
      else bin += '0';
      val /= 2;
    }
    reverse ( bin . begin ( ), bin . end ( ) );
    cout << bin << " ";
  }
}